src/lib/jsonFetcher.js

```diff
@@ -1,4 +1,4 @@
-var jQuery = ($) ? $ : require('jquery') // funny little hack, sometimes it's in the global scope, sometimes it's maybe not...
+var jQuery = (typeof $ !== 'undefined') ? $ : require('jquery') // funny little hack, sometimes it's in the global scope, sometimes it's maybe not...
```

Comment:
With two jqueries it can create some funky weirdness, particularly in testing. Oh, specifically with $.ajax, I remember some funkiness. With the sizzle selector stuff, it's totally fine. But there's somekind of singleton funkyness going on, and ajax callbacks didn't fire under specific curcumstances.